bundles/org.openhab.binding.pjlinkdevice/src/main/java/org/openhab/binding/pjlinkdevice/internal/device/command/input/InputListQueryResponse.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.pjlinkdevice.internal.device.command.input;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.openhab.binding.pjlinkdevice.internal.device.command.ErrorCode;
+import org.openhab.binding.pjlinkdevice.internal.device.command.PrefixedResponse;
+import org.openhab.binding.pjlinkdevice.internal.device.command.ResponseException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The response part of {@link InputListQueryCommand}
+ *
+ * @author Nils Schnabel - Initial contribution
+ */
+@NonNullByDefault
+public class InputListQueryResponse extends PrefixedResponse<Set<Input>> {
+  private final static HashSet<ErrorCode> SPECIFIED_ERRORCODES = new HashSet<ErrorCode>(
```

Comment:
```suggestion
  private final static HashSet<ErrorCode> SPECIFIED_ERRORCODES = new HashSet<>(
```

please use diamond operator where possible (also applies to other occurences)